writing/2016/tacl_colors.tex

```diff
@@ -557,13 +580,14 @@ \subsection{Base speaker}\label{sec:s0}
 and an affine transformation and softmax function are applied to the output to
 produce a probability distribution predicting the following token of the description.
 The model is substantively similar to well-known models for image caption generation
-\cite{Karpathy2015,Vinyals2015}, but using a second recurrent neural network for producing
+\cite{Karpathy2015,Vinyals2015}, in using a second recurrent neural network for producing
```

Comment:
This clause specifies the differences from the image captioning models, not the similarities--the image captioning models cited all use convolutional neural nets for the input, and feed in the input representation only at the beginning. We could probably just delete the "but..." if it feels like a liability to you.